.gitignore

```
@@ -7,3 +7,4 @@
 /bin
 /dist
 **/*.swp
+/.idea
```

Is this a standard directory? Why is it required?
```suggestion
/.idea

```

---

It's directory for GoLand settings, which is pretty popular IDE for GoLang, so I thing this change might be useful for many people

---

moved to another PR: https://github.com/kanisterio/kanister/pull/2376